lib/db/dbmi_base/msvc/dirent.h

```diff
@@ -0,0 +1,50 @@
+#ifndef DIRENT_INCLUDED
```

Comment:
This portability change is probably not Cmake related and could be separated out?